do/common.go

```diff
@@ -0,0 +1,53 @@
+package do
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/digitalocean/godo"
+	"github.com/digitalocean/godo/context"
+)
+
+const apiPerPage = 100
+
+func allDropletList(ctx context.Context, client *godo.Client) ([]godo.Droplet, error) {
+	// create a list to hold our droplets
+	list := []godo.Droplet{}
+
+	// create options. initially, these will be blank
```

Comment:
To be homogeneous with the rest of the project, we could probably get rid of comments that explain what we can read at code, and only leave comments that explain some internal complexity, or out of project scenarios.

As of my taste, all comments inside this function could be deleted.